<launch>
 <!-- 
    This is used to launch only the roboclaw_front node for on-robot operation.
-->
    <node name="roboclaw_front" pkg="roboclaw_driver" type="roboclaw_node.py">
        <param name="test_mode" type="bool" value="false"/>
        <param name="dev_name" type="str" value="/dev/roboclaw_front"/>
        <param name="baud" type="int" value="115200"/>
        <param name="address" type="int" value="128"/>
        <param name="loop_hz" type="int" value="4"/>
        <param name="deadman_secs" type="int" value="3"/>

        <param name="speed_cmd_topic" type="str" value="roboclaw/speed_command"/>
        <param name="stats_topic" type="str" value="roboclaw_front/stats"/>
    </node>

</launch>
